tests/system/src/test/java/test/robot/javafx/scene/ContextMenuNPETest.java

```diff
@@ -101,9 +100,8 @@ private void selectSubmenuItem() throws Exception {
             robot.keyType(KeyCode.DOWN);
             robot.keyType(KeyCode.RIGHT);
             robot.keyType(KeyCode.ENTER);
-            Toolkit.getToolkit().firePulse();
-
         });
+        Util.waitForIdle(scene);
```

Comment:
once we have waitForIdle, do we still need to wait() ?

Reply:
Same as above comment, I had kept the sleep as it is thinking if it might be required in slower systems. Since we are waiting  for 10 pulses with `waitForIdle` it might not be needed. Hence removed it.